src/openloco/localisation/stringmgr.cpp

```diff
@@ -31,37 +31,37 @@ namespace openloco::stringmgr
     static loco_global<char[NUM_USER_STRINGS][USER_STRING_SIZE], 0x0095885C> _userStrings;
 
     static std::map<int32_t, string_id> day_to_string = {
-        { 1, string_ids::day_1st },
-        { 2, string_ids::day_2nd },
-        { 3, string_ids::day_3rd },
-        { 4, string_ids::day_4th },
-        { 5, string_ids::day_5th },
-        { 6, string_ids::day_6th },
-        { 7, string_ids::day_7th },
-        { 8, string_ids::day_8th },
-        { 9, string_ids::day_9th },
-        { 10, string_ids::day_10th },
-        { 11, string_ids::day_11th },
-        { 12, string_ids::day_12th },
-        { 13, string_ids::day_13th },
-        { 14, string_ids::day_14th },
-        { 15, string_ids::day_15th },
-        { 16, string_ids::day_16th },
-        { 17, string_ids::day_17th },
-        { 18, string_ids::day_18th },
-        { 19, string_ids::day_19th },
-        { 20, string_ids::day_20th },
-        { 21, string_ids::day_21st },
-        { 22, string_ids::day_22nd },
-        { 23, string_ids::day_23rd },
-        { 24, string_ids::day_24th },
-        { 25, string_ids::day_25th },
-        { 26, string_ids::day_26th },
-        { 27, string_ids::day_27th },
-        { 28, string_ids::day_28th },
-        { 29, string_ids::day_29th },
-        { 30, string_ids::day_30th },
-        { 31, string_ids::day_31st },
+        { 0, string_ids::day_1st },
```

Comment:
We should probably just make it a normal array if it is 0-based… For now, this will do, I guess.